lib/registrant.rb

```diff
@@ -0,0 +1,19 @@
+class Registrant
+    attr_reader :name, :age, :permit, :license_data
```

Comment:
Because we have a `permit?` method defined that is intended to return the value of our instance variable `@permit`, we don't need an attr_reader for `permit`.  The attr_reader accomplishes the same thing as the method we defined, so its a bit of duplication.